README.md

```diff
@@ -183,15 +193,92 @@ Settings:
+
+- By default the `puppet.installType` is set to `agent`, and allows you to choose to use `pdk` or `agent`.
+
+- The `puppet.puppetAgentDir` allows you to choose a custom path to either a PDK install or a Puppet-Agent install, and must match the correct setting in `puppet.installType`.
+
+If no settings are chosen, the extension will attempt to use the Puppet Agent, to force it to use the PDK you have to select `pdk` for `puppet.installType`.
+
+
+**Note**: The `puppet.puppetAgentDir` setting name will be changed in the `0.12.0` release to `puppet.installDirectory`
+
+**Note**: Only PDK v1.5.0 and above are supported.
+
+### Configure extension to use PDK
+
+To ensure that the extension uses the PDK, set the `puppet.installType` setting to the `pdk` value like so:
+
+```json
+{
+  "puppet.installType":"pdk"
+}
+```
+
+The default installation paths stored in the extension are:
+
+```json
+{
+  // Windows
+  "puppet.puppetAgentDir":"C:\\Program Files\\Puppet Labs\\DevelopmentKit",
+  // Mac
+  "puppet.puppetAgentDir":"/opt/puppetlabs/pdk",
+  // Linux
+  "puppet.puppetAgentDir":"/opt/puppetlabs/pdk"
+}
+```
+
+To use a custom install path for the PDK, set the `puppet.puppetAgentDir` setting to the path you installed the PDK to:
+
+```json
+{
+  "puppet.puppetAgentDir":"D:/programs/pdk"
+}
+```
+
+### Configure extension to use Puppet-Agent
+
+To ensure that the extension uses the Puppet-Agent, set the `puppet.installType` setting to the `puppet` value like so:
+
+```json
+{
+  "puppet.installType":"puppet"
+}
+```
+
+The default installation paths stored in the extension are:
+
+```json
+{
+  // Windows
+  "puppet.puppetAgentDir":"C:\\Program Files\\Puppet Labs\\Puppet",
+  // Mac
+  "puppet.puppetAgentDir":"/opt/puppetlabs",
+  //Linux
+  "puppet.puppetAgentDir":"/opt/puppetlabs"
+}
+```
+
+To use a custom install path for the Puppet-Agent, set the `puppet.puppetAgentDir` setting to the path you installed the Puppet-Agent to:
+
+```json
+{
+  "puppet.puppetAgentDir":"D:/programs/puppet"
+}
+```
 
 ## Experience a Problem?
 
 ### Puppet Agent Install
 
-A commonly encountered problem is not having Puppet Agent installed on the computer you are running VSCode on. As noted in the [Requirements section](https://github.com/lingua-pupuli/puppet-vscode/blob/master/README.md#requirements), you will need to have Puppet Agent installed in order to fully use this extension, as the extension uses the Puppet binaries and the Ruby language bundled into the agent install in order to function.
+A commonly encountered problem is not having the PDK or Puppet Agent installed on the computer you are running VSCode on. As noted in the [Requirements section](https://github.com/lingua-pupuli/puppet-vscode/blob/master/README.md#requirements), you will need to have the PDK or Puppet Agent installed in order to fully use this extension, as the extension uses the Puppet binaries and the Ruby language bundled into the pdk or agent install in order to function.
```

Comment:
`pdk` on last line should be capitalized.

Reply:
👍